auslib/db.py

```diff
@@ -2649,6 +2659,9 @@ def hasRole(self, *args, **kwargs):
     def getUserRoles(self, *args, **kwargs):
         return self.permissions.getUserRoles(*args, **kwargs)
 
+    def getRoleUsers(self, *args, **kwargs):
```

Comment:
No need to add this - the only reason getUserRoles exists is because we need a non-table specific way to find which Roles a specific User has. We don't need that for getting the lists of all Roles with membership.

Reply:
This still needs to be removed - it is completely unused.